src/Test.UnitTests.Sarif/GitHelperTests.cs

```diff
@@ -187,7 +187,7 @@ public void GetTopLevel_WhenRepoPathIsToAFile()
         [Fact]
         public void GetTopLevel_WhenRepoPathIsToADirectory()
         {
-            string pathToDirectory = Path.GetDirectoryName(typeof(GitHelperTests).Assembly.Location);
+            string pathToDirectory = Environment.CurrentDirectory;
```

Comment:
>Environment.CurrentDirectory; [](http://example.com/codeflow?start=37&length=29)

pathToFile (line 181) and pathToDirecotory (line 190) are same after this change. Is it right?